src/AppInstallerCLIE2ETests/ConfigureCommand.cs

```diff
@@ -40,6 +40,7 @@ public void OneTimeTeardown()
         /// Intentionally has no settings to force a failure, but only after acquiring the module.
         /// </summary>
         [Test]
+        [Ignore("PS Gallery tests are unreliable.")]

```

Comment:
Instead of ignoring them, is it possible to mark them as not-breaking somehow? Like changing the Assert for a warning or something.

Reply:
Maybe do `Assert.Inconclusive()` if we get the specific failure we are seeing